from {{ importname }}.api.routes.root.errors import FooError
from {{ importname }}.foo import foo
from {{ importname }}.models.data import Foo


class Service:
    """Service for the root endpoint."""

    async def foo(self) -> Foo:
        """Get Foo."""

        try:
            return foo()
        except Exception as e:
            raise FooError() from e
